src/Akeneo/Pim/Structure/Bundle/Infrastructure/Job/JobParameters/ConstraintCollectionProvider/MoveAttributeMassEdit.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright 2023 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Akeneo\Pim\Structure\Bundle\Infrastructure\Job\JobParameters\ConstraintCollectionProvider;
+
+use Akeneo\Tool\Component\Batch\Job\JobInterface;
+use Akeneo\Tool\Component\Batch\Job\JobParameters\ConstraintCollectionProviderInterface;
+use Symfony\Component\Validator\Constraints\All;
+use Symfony\Component\Validator\Constraints\Collection;
+use Symfony\Component\Validator\Constraints\NotNull;
+use Symfony\Component\Validator\Constraints\Type;
+
+final class MoveAttributeMassEdit implements ConstraintCollectionProviderInterface
```

Comment:
This collection provider is for MassDeleteAttributes so for me it's her name
```suggestion
final class MassDeleteAttributes implements ConstraintCollectionProviderInterface
```

Reply:
We never delete the attribute here, we give them a new group